src/common/constants/enums.ts

```diff
@@ -1,13 +1,15 @@
-const roles = ['director_producer', 'camera_operator'] as const
-export type Role = (typeof roles)[number]
+export const roles = ['director_producer', 'camera_operator'] 
+export type Role = (typeof roles)[number];
 export function roleTypeChecker(arg: any): arg is Role {
-  return roles.includes(arg)
+  return roles.includes(arg);
```

Comment:
:question: :art: Are you using prettier? I thought we turned off semicolons

Reply:
I do use prettier and updated it with your prettier file changes. Forgot to format this file before pushing. 
Is there a way to automate formatting?

Reply:
In vscode settings, you can turn on a feature that formats on save.  That's what I use.  I also just by muscle memory at this point format like 10 time a minute

Reply:
Amazing!
Figured that was the case. First time truly focusing on formatting so not in the habit of formatting regularly (or even on commit).

Reply:
Looking at my VSCode settings.json I have the following which should be formatting on save:
```
    "gitlens.graph.layout": "editor",
    "git.confirmSync": false,
    "prettier.semi": false,
    "prettier.singleQuote": true,
    "editor.formatOnSave": true,
    "eslint.codeActionsOnSave.rules": null
```

So not sure why my Prettier is _not_ auto-formatting.